static/app/views/routeError.tsx

```diff
@@ -73,70 +76,59 @@ class RouteError extends Component<Props> {
 
-  render() {
-    // TODO(dcramer): show additional resource links
-    return (
-      <Alert type="error">
-        <Heading>
-          <span>{t('Oops! Something went wrong')}</span>
-        </Heading>
-        <p>
-          {t(`
+  // TODO(dcramer): show additional resource links
+  return (
+    <Alert type="error">
+      <Heading>{t('Oops! Something went wrong')}</Heading>
+      <p>
+        {t(`
           It looks like you've hit an issue in our client application. Don't worry though!
           We use Sentry to monitor Sentry and it's likely we're already looking into this!
           `)}
-        </p>
-        <p>{t("If you're daring, you may want to try the following:")}</p>
-        <List symbol="bullet">
-          {window && window.adblockSuspected && (
-            <ListItem>
-              {t(
-                "We detected something AdBlock-like. Try disabling it, as it's known to cause issues."
-              )}
-            </ListItem>
-          )}
+      </p>
+      <p>{t("If you're daring, you may want to try the following:")}</p>
+      <List symbol="bullet">
+        {window && window.adblockSuspected && (
           <ListItem>
-            {tct(`Give it a few seconds and [link:reload the page].`, {
-              link: (
-                <a
-                  onClick={() => {
-                    window.location.href = window.location.href;
-                  }}
-                />
-              ),
-            })}
+            {t(
+              "We detected something AdBlock-like. Try disabling it, as it's known to cause issues."
+            )}
           </ListItem>
-          <ListItem>
-            {tct(`If all else fails, [link:contact us] with more details.`, {
-              link: <a href="https://github.com/getsentry/sentry/issues/new/choose" />,
-            })}
-          </ListItem>
-        </List>
-      </Alert>
-    );
-  }
+        )}
+        <ListItem>
+          {tct(`Give it a few seconds and [link:reload the page].`, {
+            link: (
+              <a
+                onClick={() => {
+                  window.location.href = window.location.href;
+                }}
+              />
+            ),
+          })}
+        </ListItem>
+        <ListItem>
+          {tct(`If all else fails, [link:contact us] with more details.`, {
+            link: <a href="https://github.com/getsentry/sentry/issues/new/choose" />,
+          })}
+        </ListItem>
+      </List>
+    </Alert>
+  );
 }
 
-const Heading = styled('h3')`
-  display: flex;
-  align-items: center;
-
-  font-size: ${p => p.theme.headerFontSize};
-  font-weight: normal;
-
-  margin-bottom: ${space(1.5)};
+const Heading = styled('h1')`
+  font-size: ${p => p.theme.fontSizeLarge};
+  line-height: 1.4;
+  margin-bottom: ${space(1)};
 `;
 
-export default withRouter(withOrganization(withProject(RouteError)));
-export {RouteError};
+export default withRouter(withProject(RouteError));
```

Comment:
wasn't the named export better?

Reply:
It was used in a test and did not need to be used in the test. Better to not have multiple exports of the same thing.